core/station/impl/src/models/request_operation.rs

```diff
@@ -1093,27 +1093,46 @@ impl ModelValidator<ValidationError> for RequestOperation {
                 EnsureUserGroup::id_exists(&ok.input.user_group_id)?;
             }
             RequestOperation::SystemUpgrade(_) => (),
-            RequestOperation::ChangeExternalCanister(_) => (),
+            RequestOperation::ChangeExternalCanister(op) => {
+                let canister_id = op.input.canister_id;
+                EnsureExternalCanister::ensure_external_canister(canister_id)?;
+            }
             RequestOperation::ConfigureExternalCanister(op) => {
                 let canister_id = op.canister_id;
+                EnsureExternalCanister::ensure_external_canister(canister_id)?;
                 if let ConfigureExternalCanisterOperationKind::Settings(settings) = &op.kind {
                     if let Some(updated_request_policies) = &settings.request_policies {
                         ContextualModel::new(updated_request_policies.clone(), canister_id)
                             .validate()?;
                     }
                 }
             }
-            RequestOperation::FundExternalCanister(_) => (),
-            RequestOperation::MonitorExternalCanister(_) => (),
+            RequestOperation::FundExternalCanister(op) => {
+                let canister_id = op.canister_id;
+                EnsureExternalCanister::ensure_external_canister(canister_id)?;
+            }
+            RequestOperation::MonitorExternalCanister(op) => {
+                let canister_id = op.canister_id;
+                EnsureExternalCanister::ensure_external_canister(canister_id)?;
+            }
```

Comment:
@keplervital Is this also used for system canisters in practice? (This PR would break such flows.)